src/Tribe/Cron.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+class Tribe__Cron {
+
+	/**
+	 * Filters the cron schedules to add intervals we need.
+	 *
+	 * @param array $schedules
+	 *
+	 * @return array
+	 */
+	public function filter_cron_schedules( array $schedules ) {
+		$schedules[ 'thirty_seconds' ] = array(
+			'interval' => 30,
```

Comment:
As a question, you know that by default WordPress won't allow us to do that right? It only allows us to do I think (dont quote me on that) 15 minutes, but we can _ "force"_ it.